SPECS-EXTENDED/fstrm/fstrm.signatures.json

```diff
@@ -0,0 +1,5 @@
+{
+ "Signatures": {
+  "fstrm-v0.6.1.tar.gzz": "4f0f7ad2b760119c441aba58271e84de683b3cc138530d02710896641866e2d2"
```

Comment:
Small typo, please remove extra z in gz

I get a different hash:
bca4ac1e982a2d923ccd24cce2c98f4ceeed5009694430f73fc0dcebca8f098f  fstrm-0.6.1.tar.gz